codecov_auth/migrations/0041_owner_trial_fired_by_alter_user_terms_agreement.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.3 on 2023-09-18 09:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("codecov_auth", "0040_oktauser"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="owner",
+            name="trial_fired_by",
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AlterField(
```

Comment:
Huuuuh, what's this doing here 👀 is this part of your PR?

Reply:
it shouldn't be, but it looks like the last migration applied to this field didn't capture "default":
`field=models.BooleanField(null=True),
`
it should be:
`
field=models.BooleanField(default=False, null=True),`

Reply:
or i have no idea

Reply:
synced w/@jerry it's safe not to include it